src/main/webapp/app/overview/courses-routing.module.ts

```diff
@@ -189,7 +189,9 @@ const routes: Routes = [
             },
             {
                 path: 'learning-path',
-                loadChildren: () => import('app/course/learning-paths/learning-paths.module').then((m) => m.ArtemisLearningPathsModule),
+                loadComponent: () =>
+                    import('app/course/learning-paths/pages/learning-path-student-page/learning-path-student-page.component').then((c) => c.LearningPathStudentPageComponent),
+                // loadChildren: () => import('app/course/learning-paths/learning-paths.module').then((m) => m.ArtemisLearningPathsModule),
```

Comment:
Do you want to keep this outcommented?

Reply:
Thanks for the hint, we missed that comment